overlord/devicestate/devicestate_install_mode_test.go

```diff
@@ -215,6 +216,9 @@ func (s *deviceMgrInstallModeSuite) doRunChangeTestWithEncryption(c *C, grade st
 	c.Assert(modeenv.WriteTo(""), IsNil)
 	devicestate.SetSystemMode(s.mgr, "install")
 
+	err := os.MkdirAll(boot.InitramfsUbuntuBootDir, 0755)
```

Comment:
for realism if I understand correctly, this should be done from inside the mocked snap-boostrap

Reply:
Yes. I'll leave a comment here.